tests/test_minhash.py

```diff
@@ -1792,6 +1797,9 @@ def test_merge_abund():
     ret = mh1.merge(mh2)
     assert ret is None
 
+    ret= mh1 | mh2
```

Comment:
so, `merge` here turns out to be the wrong method to use :).

see https://github.com/dib-lab/sourmash/issues/1211 and https://github.com/dib-lab/sourmash/pull/1282 for some discussion, but the short version is this: define `__or__` to be equivalent to `__add__` instead of `merge`.

* `merge` modifies the first object, so, above, `mh1` is updated with the contents of `mh2`. This is called 'in place' modification, because the object that is calling the method is updated.
* the `|` operator usually returns a _new_ object that contains the new result. This is similar to the behavior that (e.g.) `+` has, vs `+=` - the former is a standard operator, the latter is `in place`.
* so you 

You can read more at https://www.geeksforgeeks.org/inplace-vs-standard-operators-python/